app/views/application/_header.html.erb

```diff
@@ -10,7 +10,7 @@
       <%= link_to "On Fire", on_fire_path, class: "navigation__item #{ "navigation__item--active" if controller_name == "on_fire" }" %>
       <%= link_to "Wiki", wiki_root_path, class: "navigation__item #{ "navigation__item--active" if is_wiki? }" %>
       <%= link_to "Editor", editor_path, class: "navigation__item hidden visible-lg", target: "_blank" %>
-      <%= link_to inline_svg_tag("discord.svg"), "https://discord.gg/DdxsQRD", target: "_blank", rel: "noreferrer noopener", class: "navigation__item mt-1/4 mbl:mt-1/8", aria: { label: "Join the Workhsop.codes Discord" } %>
+      <%= link_to inline_svg_tag("icons/icon-discord.svg", width: 25, height: 25), "https://discord.gg/DdxsQRD", target: "_blank", rel: "noreferrer noopener", style: "display: flex", class: "navigation__item", aria: { label: "Join the Workhsop.codes Discord" } %>
```

Comment:
Instead of the inline style, there is a `flex` utility class (why is this class necessary anyway?)

```suggestion
      <%= link_to inline_svg_tag("icons/icon-discord.svg", width: 25, height: 25), "https://discord.gg/DdxsQRD", target: "_blank", rel: "noreferrer noopener", class: "navigation__item flex", aria: { label: "Join the Workhsop.codes Discord" } %>
```

Reply:
The flexx CSS class also has 100% width for some reason so i couldn't use it. It just centers the icon. The one before used some margins to try and center it but just setting it to flex does it automatically.

Reply:
Ah ok, that's annoying

Reply:
I updated it to use a class, I forgot i also needed to add the `text-discord` on the icon so I figured it might as well just use its own navigation item class. I'm not 100% sure how it works in practice as the navigation items color can change depending on hover or the page your on and i'm unsure if that would override the color of the icon.

Reply:
I think it would, but that's ok, it needs a hover state after all

Reply:
It already has the underline from the nav item. I think maybe i'll just use `!important` to keep it blue, hoving turns it white at the moment which I think is a bit weird. Though the wiki page shows all the text as white so you could also make the icon white on the wiki page if you wanted.

Reply:
Let's not use !important, that will be annoying in the future. If you want it to stay purple you can overwrite the hover state

```
&:hover {
  color: $discord;
}
```